Packages/com.unity.inputsystem/InputSystem/Editor/PropertyDrawers/GpadButtonPropertyDrawer.cs

```diff
@@ -0,0 +1,82 @@
+#if UNITY_EDITOR
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine.InputSystem.LowLevel;
+using UnityEditor;
+using UnityEngine.UIElements;
+
+namespace UnityEngine.InputSystem.Editor
+{
+    /// <summary>
+    /// Property drawer for <see cref = "GamepadButton" />
+    /// </summary >
+    [CustomPropertyDrawer(typeof(GamepadButton))]
+    internal class GpadButtonPropertyDrawer : PropertyDrawer
+    {
+        private string[] m_EnumDisplayNames;
+
+        public override VisualElement CreatePropertyGUI(SerializedProperty property)
+        {
+            var enumNamesAndValues = new Dictionary<string, int>();
+            var enumDisplayNames = Enum.GetNames(typeof(GamepadButton));
+            var enumValues = Enum.GetValues(typeof(GamepadButton)).Cast<GamepadButton>().ToArray();
+
+            for (var i = 0; i < enumDisplayNames.Length; ++i)
+            {
+                string enumName = enumDisplayNames[i];
```

Comment:
This assignment could be on line 40 right since overwritten for the special cases?